snuba/consumers/consumer_builder.py

```diff
@@ -109,29 +129,42 @@ def __init__(
 
         stream_loader = self.storage.get_table_writer().get_stream_loader()
 
-        self.raw_topic: Topic
+        self.raw_topic: ArroyoTopic
```

Comment:
Personal preference, I'd rather you just left all these names.

Reply:
The issue with leaving the names is the potential confusion between (Snuba) `Topic` and an (Arroyo) `Topic`. If both are `Topic` (how it was originally), it can become confusing to work with the code in future iterations.

Reply:
I think the issue is the other topic (SnubaTopic) is being imported here but that should never be the case. It should always be accessed via the stream loader and KafkaTopicSpec mechanism to ensure that topic resolving is done correctly.

Reply:
This makes sense. I no longer have those imports. Shall I change `ArroyoTopic` back to `Topic`?

Reply:
That seems good to me. It feels a little bit more consistent with what the other files do.

Reply:
Bumping this. Can you please revert the change?